ososedki_dl/crawlers/other/wildskirts.py

```
@@ -47,26 +49,98 @@ def wildskirts_media_filter(self, soup: BeautifulSoup) -> list[str]:
-            url (str): The URL of the page to extract media URLs from.
+        profile_id: str = profile_id_input["value"]  # type: ignore
+        api_url: str = f"{self.api_url}/{profile_id}"
+        logger.debug(f"Fetching media URLs from API endpoint: {api_url}")
+
+        try:
+            data = await self.downloader.fetch(api_url, response_property="json")
+            media_items = data.get("media", {}).get("items", {})
+            urls = [item.get("u") for item in media_items.values() if item.get("u")]
+            logger.debug(f"Extracted {len(urls)} media URLs from API response")
+            return urls
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)